gdal/configure.ac

```diff
@@ -5802,6 +5949,10 @@ LOC_MSG([  MongoDB support:           ${MONGODB_ENABLED}])
 LOC_MSG([  MongoCXX v3 support:       ${MONGOCXXV3_ENABLED}])
 LOC_MSG([  HDFS support:              ${HDFS_ENABLED}])
 LOC_MSG([  userfaultfd support:       ${ENABLE_UFFD}])
+LOC_MSG([  misc. gdal formats:       ${GDALFORMATS_ENABLED}])
```

Comment:
missing space character for alignment

Reply:
ignore this remark. Just a diff artifact. The output of https://travis-ci.com/OSGeo/gdal/jobs/174719812 looks good

Reply:
actually it's not a diff artifact, I removed a space as GDAL_FORMATS_ENABLED will start with a space due to how it is constructed